android/java/res/layout/fragment_secure_password.xml

```diff
@@ -25,20 +24,17 @@
             android:layout_marginBottom="18dp"
             android:text="@string/create_password"
             android:textColor="@color/wallet_text_color"
-            android:lineSpacingExtra="2sp"
-            android:textSize="22sp"
-            />
+            android:textStyle="bold" />
```

Comment:
@simoarpe do we need to add textStyle here ?

Reply:
I've tried without the bold attribute, but the end result appeared substantially different from the Figma design provided.